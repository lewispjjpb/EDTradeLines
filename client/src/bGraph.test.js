import React from 'react';
import renderer from 'react-test-renderer';

import BuyGraph from './bGraph.jsx';


const data = [['test value']];

const thing = () => 'test value'

test('two plus two is four', () => {
  expect(2 + 2).toBe(4);
});

test('values to match', () => {
  const component = renderer.create(
    <BuyGraph />
  )

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

  tree.props.testClick('input')

  // expect(tree.props.testClick('the test').toBe('the test'));
  tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});

