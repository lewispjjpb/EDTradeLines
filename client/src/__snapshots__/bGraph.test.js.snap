// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`values to match 1`] = `
<div
  testClick={[Function]}
>
  <div
    style={
      Object {
        "display": "inline-block",
        "position": "relative",
      }
    }
  />
</div>
`;

exports[`values to match 2`] = `
<div
  testClick={[Function]}
>
  <div
    style={
      Object {
        "display": "inline-block",
        "position": "relative",
      }
    }
  />
</div>
`;
